hooks/post_gen_project.py

```diff
@@ -50,6 +50,21 @@ def set_up_framework_and_tests():
             shutil.move(join(testing_folder, file_or_folder), '.')
 
 
+def prune_cronjob_version(deployment_folder):
```

Comment:
Is this really a "version" or rather a "style", or something else?

Reply:
true, the naming is not quite right.